distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -121,7 +121,7 @@ grant codeBase "file:/org.apache.servicemix.bundles.wsdl4j" {
     permission java.io.FilePermission "<<ALL FILES>>", "read";
 }
 
-grant codeBase "file:/org.apache.karaf.shell.core/registry-federation-admin-service-impl/catalog-core-commands/spatial-geocoding-offline-catalog/spatial-geocoding-geocoder/spatial-geocoding-plugin/spatial-wfs-v1_0_0-source/spatial-wfs-v2_0_0-source/spatial-wfs-v1_1_0-source/spatial-wfs-featuretransformer/spatial-wfs-featuretransformer-xstream/resourcemanagement-usage-ui" {
+grant codeBase "file:/org.apache.karaf.shell.core/registry-federation-admin-service-impl/catalog-core-commands/spatial-geocoding-offline-catalog/spatial-geocoding-geocoder/spatial-geocoding-plugin/spatial-wfs-v1_0_0-source/spatial-wfs-v2_0_0-source/spatial-wfs-v1_1_0-source/spatial-wfs-featuretransformer/spatial-wfs-featuretransformer-xstream/resourcemanagement-usage-ui/catalog-rest-endpoint" {
```

Comment:
❓was this just a name change on the bundle or was it not being specific enough? Want to clarify that the former ```resourcemanagement-usage-ui``` won't need those permissions for anything

Reply:
No that was my bad not understanding it fully, I thought it was a large filepath. after clarifying with @bakejeyner looks fine :+1: